Release 55.0.0

brad-decker opened the pull request:
## Description
See changelog for details

mcmire commented on packages/assets-controllers/CHANGELOG.md:
```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
+## [9.0.0]
+### Added
+- **BREAKING**: Add multi account balances fetch flag support ([#1146](https://github.com/MetaMask/core.git/pull/1146))
```
Hmm. It appears that all of these new changelog entries are using `https://github.com/MetaMask/core.git` for the repo URL, but it actually needs to be `https://github.com/MetaMask/core`. I'll have to look into why this is happening. In the meantime I'll add some suggestions for replacing these. This should get CI to pass the "validate changelog" step.

mcmire replied:
Also maybe we flesh this out a bit based on the changelog entries listed in this PR:

``` suggestion
- **BREAKING**: Add required options `getSelectedAddress` and `getMultiAccountBalancesEnabled` to AccountTrackerController constructor and make use of them when refreshing account balances ([#1146](https://github.com/MetaMask/core/pull/1146))
  - Previously, the controller would refresh all account balances, but these options can be used to only refresh the currently selected account

mcmire commented on packages/assets-controllers/CHANGELOG.md:
```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+## [9.0.0]
+### Added
+- **BREAKING**: Add multi account balances fetch flag support ([#1146](https://github.com/MetaMask/core.git/pull/1146))
+- Add ERC721 & ERC1155 `wallet_watchAsset` support ([#1173](https://github.com/MetaMask/core.git/pull/1173))
```
``` suggestion
- Add `watchNft` and `validateWatchNft` methods to NftController, which can be used to watch an ERC-721 or ERC-1155 token ([#1173](https://github.com/MetaMask/core.git/pull/1173))

mcmire commented on packages/assets-controllers/CHANGELOG.md:
```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Add ERC721 & ERC1155 `wallet_watchAsset` support ([#1173](https://github.com/MetaMask/core.git/pull/1173))
+
+### Changed
+- fix method description ([#1401](https://github.com/MetaMask/core.git/pull/1401))
```
This isn't user-facing, so we can drop this. However, in its place I think we should note two changes in dependencies:

``` suggestion
- Add dependency on `@ethersproject/address` ([#1173](https://github.com/MetaMask/core.git/pull/1173))
- Replace `eth-rpc-errors` with `@metamask/rpc-errors` ([#1173](https://github.com/MetaMask/core.git/pull/1173))

mcmire commented on packages/keyring-controller/CHANGELOG.md:
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
+## [5.1.0]
+### Added
+- Add cancelQRSynchronization method ([#1387](https://github.com/MetaMask/core.git/pull/1387))
```
Just a tweak:

``` suggestion
- Add `cancelQRSynchronization` method ([#1387](https://github.com/MetaMask/core.git/pull/1387))

mcmire commented on packages/network-controller/CHANGELOG.md:
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
+## [10.0.0]
+### Changed
+- Update getEIP1559Compatibility to match extension ([#1404](https://github.com/MetaMask/core.git/pull/1404))
```
We can expand this a bit:

``` suggestion
- **BREAKING:** Update `getEIP1559Compatibility` to return `false` instead of `true` if the provider has not been initialized yet ([#1404](https://github.com/MetaMask/core/pull/1404))
- Update `getEIP1559Compatibility` to not hit the current network if it is known that it does not support EIP-1559 ([#1404](https://github.com/MetaMask/core/pull/1404))
- Update `networkDetails` initial state from `{ EIPS: { 1559: false } }` to `{ EIPS: {} }` ([#1404](https://github.com/MetaMask/core/pull/1404))

mcmire commented on packages/network-controller/CHANGELOG.md:
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+## [10.0.0]
+### Changed
+- Update getEIP1559Compatibility to match extension ([#1404](https://github.com/MetaMask/core.git/pull/1404))
+- Reset network details when lookupNetwork request fails ([#1403](https://github.com/MetaMask/core.git/pull/1403))
```
``` suggestion
Update lookupNetwork to unset `networkDetails.EIPS[1559]` in state instead of setting it `false` if either of its requests for the network ID or network details fails ([#1403](https://github.com/MetaMask/core/pull/1403))

mcmire commented on packages/network-controller/CHANGELOG.md:
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+### Changed
+- Update getEIP1559Compatibility to match extension ([#1404](https://github.com/MetaMask/core.git/pull/1404))
+- Reset network details when lookupNetwork request fails ([#1403](https://github.com/MetaMask/core.git/pull/1403))
+- Remove fake timers from NetworkController tests ([#1399](https://github.com/MetaMask/core.git/pull/1399))
+- Use Jest snapshots for testing initial NetworkController state ([#1397](https://github.com/MetaMask/core.git/pull/1397))
```
We can drop these since they're just changes to tests and shouldn't be visible to end users:

``` suggestion

mcmire commented on packages/network-controller/CHANGELOG.md:
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.0]
```
This turns out to be correct since there was a breaking change, so, 👍🏻

mcmire commented on packages/preferences-controller/CHANGELOG.md:
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.0.0]
+### Added
+- **BREAKING**: Add multi account balances fetch flag support ([#1146](https://github.com/MetaMask/core.git/pull/1146))
```
It looks like the change to PreferencesController isn't breaking, just an addition:

``` suggestion
## [4.1.0]
### Added
- Add `isMultiAccountBalancesEnabled` to state (default: true) along with a `setIsMultiAccountBalancesEnabled` method to set it
```

mcmire commented on packages/preferences-controller/CHANGELOG.md:
```diff
@@ -42,11 +46,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-[1.0.2]: https://github.com/MetaMask/core/compare/@metamask/preferences-controller@1.0.1...@metamask/preferences-controller@1.0.2
-[1.0.1]: https://github.com/MetaMask/core/compare/@metamask/preferences-controller@1.0.0...@metamask/preferences-controller@1.0.1
-[1.0.0]: https://github.com/MetaMask/core/releases/tag/@metamask/preferences-controller@1.0.0
+[Unreleased]: https://github.com/MetaMask/core.git/compare/@metamask/preferences-controller@5.0.0...HEAD
+[5.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/preferences-controller@4.0.0...@metamask/preferences-controller@5.0.0
+[4.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/preferences-controller@3.0.0...@metamask/preferences-controller@4.0.0
+[3.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/preferences-controller@2.1.0...@metamask/preferences-controller@3.0.0
+[2.1.0]: https://github.com/MetaMask/core.git/compare/@metamask/preferences-controller@2.0.0...@metamask/preferences-controller@2.1.0
+[2.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/preferences-controller@1.0.2...@metamask/preferences-controller@2.0.0
+[1.0.2]: https://github.com/MetaMask/core.git/compare/@metamask/preferences-controller@1.0.1...@metamask/preferences-controller@1.0.2
+[1.0.1]: https://github.com/MetaMask/core.git/compare/@metamask/preferences-controller@1.0.0...@metamask/preferences-controller@1.0.1
+[1.0.0]: https://github.com/MetaMask/core.git/releases/tag/@metamask/preferences-controller@1.0.0
```
``` suggestion
[Unreleased]: https://github.com/MetaMask/core/compare/@metamask/preferences-controller@4.1.0...HEAD
[4.1.0]: https://github.com/MetaMask/core/compare/@metamask/preferences-controller@4.0.0...@metamask/preferences-controller@4.1.0
[4.0.0]: https://github.com/MetaMask/core/compare/@metamask/preferences-controller@3.0.0...@metamask/preferences-controller@4.0.0
[3.0.0]: https://github.com/MetaMask/core/compare/@metamask/preferences-controller@2.1.0...@metamask/preferences-controller@3.0.0
[2.1.0]: https://github.com/MetaMask/core/compare/@metamask/preferences-controller@2.0.0...@metamask/preferences-controller@2.1.0
[2.0.0]: https://github.com/MetaMask/core/compare/@metamask/preferences-controller@1.0.2...@metamask/preferences-controller@2.0.0
[1.0.2]: https://github.com/MetaMask/core/compare/@metamask/preferences-controller@1.0.1...@metamask/preferences-controller@1.0.2
[1.0.1]: https://github.com/MetaMask/core/compare/@metamask/preferences-controller@1.0.0...@metamask/preferences-controller@1.0.1
[1.0.0]: https://github.com/MetaMask/core/releases/tag/@metamask/preferences-controller@1.0.0
```

mcmire commented on packages/preferences-controller/package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/preferences-controller",
-  "version": "4.0.0",
+  "version": "5.0.0",
```
``` suggestion
  "version": "4.1.0",
```

mcmire commented on packages/assets-controllers/CHANGELOG.md:
```diff
@@ -148,17 +156,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-[2.0.0]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@1.0.1...@metamask/assets-controllers@2.0.0
-[1.0.1]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@1.0.0...@metamask/assets-controllers@1.0.1
-[1.0.0]: https://github.com/MetaMask/core/releases/tag/@metamask/assets-controllers@1.0.0
+[Unreleased]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@9.0.0...HEAD
+[9.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@8.0.0...@metamask/assets-controllers@9.0.0
+[8.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@7.0.0...@metamask/assets-controllers@8.0.0
+[7.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@6.0.0...@metamask/assets-controllers@7.0.0
+[6.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@5.1.0...@metamask/assets-controllers@6.0.0
+[5.1.0]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@5.0.1...@metamask/assets-controllers@5.1.0
+[5.0.1]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@5.0.0...@metamask/assets-controllers@5.0.1
+[5.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@4.0.1...@metamask/assets-controllers@5.0.0
+[4.0.1]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@4.0.0...@metamask/assets-controllers@4.0.1
+[4.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@3.0.1...@metamask/assets-controllers@4.0.0
+[3.0.1]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@3.0.0...@metamask/assets-controllers@3.0.1
+[3.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@2.0.0...@metamask/assets-controllers@3.0.0
+[2.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@1.0.1...@metamask/assets-controllers@2.0.0
+[1.0.1]: https://github.com/MetaMask/core.git/compare/@metamask/assets-controllers@1.0.0...@metamask/assets-controllers@1.0.1
+[1.0.0]: https://github.com/MetaMask/core.git/releases/tag/@metamask/assets-controllers@1.0.0
```
``` suggestion
[Unreleased]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@9.0.0...HEAD
[9.0.0]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@8.0.0...@metamask/assets-controllers@9.0.0
[8.0.0]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@7.0.0...@metamask/assets-controllers@8.0.0
[7.0.0]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@6.0.0...@metamask/assets-controllers@7.0.0
[6.0.0]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@5.1.0...@metamask/assets-controllers@6.0.0
[5.1.0]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@5.0.1...@metamask/assets-controllers@5.1.0
[5.0.1]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@5.0.0...@metamask/assets-controllers@5.0.1
[5.0.0]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@4.0.1...@metamask/assets-controllers@5.0.0
[4.0.1]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@4.0.0...@metamask/assets-controllers@4.0.1
[4.0.0]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@3.0.1...@metamask/assets-controllers@4.0.0
[3.0.1]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@3.0.0...@metamask/assets-controllers@3.0.1
[3.0.0]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@2.0.0...@metamask/assets-controllers@3.0.0
[2.0.0]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@1.0.1...@metamask/assets-controllers@2.0.0
[1.0.1]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@1.0.0...@metamask/assets-controllers@1.0.1
[1.0.0]: https://github.com/MetaMask/core/releases/tag/@metamask/assets-controllers@1.0.0

mcmire commented on packages/keyring-controller/CHANGELOG.md:
```diff
@@ -74,10 +78,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-[2.0.0]: https://github.com/MetaMask/core/compare/@metamask/keyring-controller@1.0.1...@metamask/keyring-controller@2.0.0
-[1.0.1]: https://github.com/MetaMask/core/compare/@metamask/keyring-controller@1.0.0...@metamask/keyring-controller@1.0.1
-[1.0.0]: https://github.com/MetaMask/core/releases/tag/@metamask/keyring-controller@1.0.0
+[Unreleased]: https://github.com/MetaMask/core.git/compare/@metamask/keyring-controller@6.0.0...HEAD
+[6.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/keyring-controller@5.0.0...@metamask/keyring-controller@6.0.0
+[5.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/keyring-controller@4.0.0...@metamask/keyring-controller@5.0.0
+[4.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/keyring-controller@3.0.0...@metamask/keyring-controller@4.0.0
+[3.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/keyring-controller@2.0.0...@metamask/keyring-controller@3.0.0
+[2.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/keyring-controller@1.0.1...@metamask/keyring-controller@2.0.0
+[1.0.1]: https://github.com/MetaMask/core.git/compare/@metamask/keyring-controller@1.0.0...@metamask/keyring-controller@1.0.1
+[1.0.0]: https://github.com/MetaMask/core.git/releases/tag/@metamask/keyring-controller@1.0.0
```
``` suggestion
[Unreleased]: https://github.com/MetaMask/core/compare/@metamask/keyring-controller@6.0.0...HEAD
[6.0.0]: https://github.com/MetaMask/core/compare/@metamask/keyring-controller@5.0.0...@metamask/keyring-controller@6.0.0
[5.0.0]: https://github.com/MetaMask/core/compare/@metamask/keyring-controller@4.0.0...@metamask/keyring-controller@5.0.0
[4.0.0]: https://github.com/MetaMask/core/compare/@metamask/keyring-controller@3.0.0...@metamask/keyring-controller@4.0.0
[3.0.0]: https://github.com/MetaMask/core/compare/@metamask/keyring-controller@2.0.0...@metamask/keyring-controller@3.0.0
[2.0.0]: https://github.com/MetaMask/core/compare/@metamask/keyring-controller@1.0.1...@metamask/keyring-controller@2.0.0
[1.0.1]: https://github.com/MetaMask/core/compare/@metamask/keyring-controller@1.0.0...@metamask/keyring-controller@1.0.1
[1.0.0]: https://github.com/MetaMask/core/releases/tag/@metamask/keyring-controller@1.0.0

mcmire commented on packages/logging-controller/CHANGELOG.md:
```diff
@@ -6,4 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Initial Release
+  - Add logging controller ([#1089](https://github.com/MetaMask/core.git/pull/1089))
+
+[Unreleased]: https://github.com/MetaMask/core.git/compare/@metamask/logging-controller@1.0.0...HEAD
+[1.0.0]: https://github.com/MetaMask/core.git/releases/tag/@metamask/logging-controller@1.0.0
```
``` suggestion
[Unreleased]: https://github.com/MetaMask/core/compare/@metamask/logging-controller@1.0.0...HEAD
[1.0.0]: https://github.com/MetaMask/core/releases/tag/@metamask/logging-controller@1.0.0

mcmire commented on packages/network-controller/CHANGELOG.md:
```diff
@@ -173,13 +180,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-[3.0.0]: https://github.com/MetaMask/core/compare/@metamask/network-controller@2.0.0...@metamask/network-controller@3.0.0
-[2.0.0]: https://github.com/MetaMask/core/compare/@metamask/network-controller@1.0.0...@metamask/network-controller@2.0.0
-[1.0.0]: https://github.com/MetaMask/core/releases/tag/@metamask/network-controller@1.0.0
+[Unreleased]: https://github.com/MetaMask/core.git/compare/@metamask/network-controller@10.0.0...HEAD
+[10.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/network-controller@9.0.0...@metamask/network-controller@10.0.0
+[9.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/network-controller@8.0.0...@metamask/network-controller@9.0.0
+[8.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/network-controller@7.0.0...@metamask/network-controller@8.0.0
+[7.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/network-controller@6.0.0...@metamask/network-controller@7.0.0
+[6.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/network-controller@5.0.0...@metamask/network-controller@6.0.0
+[5.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/network-controller@4.0.0...@metamask/network-controller@5.0.0
+[4.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/network-controller@3.0.0...@metamask/network-controller@4.0.0
+[3.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/network-controller@2.0.0...@metamask/network-controller@3.0.0
+[2.0.0]: https://github.com/MetaMask/core.git/compare/@metamask/network-controller@1.0.0...@metamask/network-controller@2.0.0
+[1.0.0]: https://github.com/MetaMask/core.git/releases/tag/@metamask/network-controller@1.0.0
```
``` suggestion
[Unreleased]: https://github.com/MetaMask/core/compare/@metamask/network-controller@10.0.0...HEAD
[10.0.0]: https://github.com/MetaMask/core/compare/@metamask/network-controller@9.0.0...@metamask/network-controller@10.0.0
[9.0.0]: https://github.com/MetaMask/core/compare/@metamask/network-controller@8.0.0...@metamask/network-controller@9.0.0
[8.0.0]: https://github.com/MetaMask/core/compare/@metamask/network-controller@7.0.0...@metamask/network-controller@8.0.0
[7.0.0]: https://github.com/MetaMask/core/compare/@metamask/network-controller@6.0.0...@metamask/network-controller@7.0.0
[6.0.0]: https://github.com/MetaMask/core/compare/@metamask/network-controller@5.0.0...@metamask/network-controller@6.0.0
[5.0.0]: https://github.com/MetaMask/core/compare/@metamask/network-controller@4.0.0...@metamask/network-controller@5.0.0
[4.0.0]: https://github.com/MetaMask/core/compare/@metamask/network-controller@3.0.0...@metamask/network-controller@4.0.0
[3.0.0]: https://github.com/MetaMask/core/compare/@metamask/network-controller@2.0.0...@metamask/network-controller@3.0.0
[2.0.0]: https://github.com/MetaMask/core/compare/@metamask/network-controller@1.0.0...@metamask/network-controller@2.0.0
[1.0.0]: https://github.com/MetaMask/core/releases/tag/@metamask/network-controller@1.0.0

mcmire reviewed (commented):
Left some comments!

brad-decker commented:
@mcmire I added all your commits, thank you! Just a note I rebased this on main and pulled in two commits, one is your changes to add a missing test for network controller so I think no changes to changelog are needed. The other is one on assets-controller from @adonesky1 who is sending me the changelog entry.

adonesky1 commented on packages/assets-controllers/CHANGELOG.md:
```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+  - The `NFTController` now requires an instance of a ControllerMessenger to be passed to its constructor. This is messenger is used to pass the `watchNFT` message to the `ApprovalController`.
+
+### Changed
+- Add dependency on `@ethersproject/address` ([#1173](https://github.com/MetaMask/core.git/pull/1173))
```
Does this need to be included in the changelog?

adonesky1 replied:
cc @Gudahtt

mcmire replied:
Historically we have been listing added/changed dependencies in the changelog (as long as they are runtime or peer dependencies, not development dependencies). For instance, see https://github.com/MetaMask/core/blob/main/packages/gas-fee-controller/CHANGELOG.md#600 or https://github.com/MetaMask/core/blob/main/packages/approval-controller/CHANGELOG.md#300). This advises consumers of changes to the dependency tree that would occur with an upgrade to this version. For us it especially lets us know when we might get new Socket alerts or when there might be new changes to LavaMoat policy files.

adonesky1 commented on packages/assets-controllers/CHANGELOG.md:
```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+
+### Changed
+- Add dependency on `@ethersproject/address` ([#1173](https://github.com/MetaMask/core.git/pull/1173))
+- Replace `eth-rpc-errors` with `@metamask/rpc-errors` ([#1173](https://github.com/MetaMask/core.git/pull/1173))
```
or this one for that matter?

mcmire commented on packages/network-controller/CHANGELOG.md:
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- **BREAKING:** Update `getEIP1559Compatibility` to return `false` instead of `true` if the provider has not been initialized yet ([#1404](https://github.com/MetaMask/core/pull/1404))
+- Update `getEIP1559Compatibility` to not hit the current network if it is known that it does not support EIP-1559 ([#1404](https://github.com/MetaMask/core/pull/1404))
+- Update `networkDetails` initial state from `{ EIPS: { 1559: false } }` to `{ EIPS: {} }` ([#1404](https://github.com/MetaMask/core/pull/1404))
+Update lookupNetwork to unset `networkDetails.EIPS[1559]` in state instead of setting it `false` if either of its requests for the network ID or network details fails ([#1403](https://github.com/MetaMask/core/pull/1403))
```
Hmm, it should have — I can check our `auto-changelog` package to see what's going on here.

adonesky1 reviewed (approved):
LGTM!
